vcd/resource_vcd_vm_internal_disk.go

```diff
@@ -0,0 +1,475 @@
+// /*****************************************************************
```

Comment:
I haven't seen us putting licenses anywhere in provider code.

Reply:
removed